src/Model/CustomField.php

```diff
@@ -13,6 +13,20 @@ class CustomField
 
     function processData($data)
     {
+
+        $data['Type'] = "";
+        if (isset($data['Date']))
+            $data['Type'] = "Date";
+        if (isset($data['Number']))
+            $data['Type'] = "Number";
+        if (isset($data['Decimal']))
+            $data['Type'] = "Decimal";
+        if (isset($data['Boolean']))
+            $data['Type'] = "Boolean";
+        if (isset($data['Text']))
+            $data['Type'] = "Text";
+
+        
         return $data;
```

Comment:
Adding `$data['Value'] = $data[$data['Type']]` would simplify the external code, I think?

Reply:
See Commits 3bfc4e7fb5a3e7b8f7177abb0eeedc68716f4b10 and ab4b17382f6289c6d5be65db3cc0b5e64e5a8d79